src/modules/bulk-minter/components/Loading.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from "react";
+
+interface LoadingProps {
+    className?: string
+    w?: string
+    h?: string
+  }
+  
+const Loading: FunctionComponent<LoadingProps> = ({
+    className,
+    w = 5,
+    h = 5,
+  }) => {
+    return (
+      <div className="flex justify-center w-full">
+        <svg
+          className={`${className} animate-spin h-${w} w-${h}`}
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+        >
+          <circle
+            className={`opacity-25`}
```

Comment:
unnecessary template literal
```suggestion
            className="opacity-25"
```